lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -501,7 +501,9 @@ lldb::FunctionSP SymbolFileNativePDB::CreateFunction(PdbCompilandSymId func_id,
     return nullptr;
 
   PdbTypeSymId sig_id(proc.FunctionType, false);
-  Mangled mangled(proc.Name);
+  auto mangled_opt =
```

Comment:
nit:
```suggestion
  std::optional<llvm::StringRef> mangled_opt =
```
Where it's not immediately obvious what the type is we usually avoid `auto`